src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="images/logo.png"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__image">
+          <a href="#">
```

Comment:
The 'href' attribute in anchor tags should lead to a valid destination. If this link is supposed to be a placeholder, consider using a '#' for now.